searx/engines/360search.py

```diff
@@ -26,22 +32,62 @@
 
 # Base URL
 base_url = "https://www.so.com"
+cache_key = f"{base_url}-cookie"
```

Comment:
```suggestion
COOKIE_CACHE_KEY = "cookie"
```
Should be a constant and doesn't have to be engine base url dependent because the `EngineCache` itself is already engine specific